### DISCUSSION TITLE
Make two record in same time

### DISCUSSION THREAD

**Discussion by laldevfr** [llfbandit/record#131] [Q&A] [answered]
Hi,

I want to build a feature in my app like : 
   - Create a main record 
   - Tape on button and then : 
       -- pause main record
       -- Launch new record
       -- Stop new record 
  -Back to main record and resume it
  -...
  - Stop main record
  
With record package it is possible to do it ?

Laurent.

**Comment by llfbandit** [comment, marked as answer]
There is no such feature provided with the package.
Only a single instance is managed.

**Comment by llfbandit** [comment]
In v5, multiple instances are supported. In beta for now.
